### ISSUE TITLE
Login prompt never shown

### ISSUE THREAD

**Issue by zubozrout** [Hankamotz/Fmessenger#12]
Hello,
when I try to use this app on a newly installed device it won't allow me to log in. Messenger just says I should login through Facebook (and if I visit Facebook via this app the login credentials are not used for the chat).

Could you please look into that if there is a way to fix this. I was really relying on this app since there is none of this kind in the store.

Thank you very much.

**Comment by Hankamotz** [comment]
Yes, I look the same issue too. In some devices the login works well (Aquaris M10 tablet for example), I think it will be related to User Agent, I hope to fix it soon.

**Comment by p-mitana** [comment]
Does not work for me too (Meizu Pro 5 Ubuntu Edition, switched to ubports stable recently). It says "Login to Facebook to start" (or similar, I translated what I had seen). Below this message is a spinner spins endlessly (I guess it's the same case as zubozrout's).

**Comment by aorzh** [comment]
Same issue on OnePlus One.

**Comment by dekkerj1** [comment]
Same issue on Nexus 4

**Comment by UmarAbduvali** [comment]
@Hankamotz 
Is there any news about log in fix? :D

**Comment by aorzh** [comment]
@UmarAbduvali , looks like project is not supported already - last commit was a year ago. I'll try to create a pull request with fixes later. Right now I'm very busy.
